hoprd/hoprd/src/main.rs

```diff
@@ -47,6 +47,32 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let subscriber = tracing_subscriber::Registry::default().with(env_filter).with(format);
 
+    let telemetry_url = Some("http://localhost:4317".to_owned());
```

Comment:
Should the url be obtained from the config variable ?

Reply:
Passed in as an env variable.